ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -96,6 +96,8 @@ function NetworkGraphPage() {
                     </ToolbarContent>
                 </Toolbar>
                 <Divider component="div" />
+            </PageSection>
```

Comment:
I think the `<Divider>` component should be between the `<PageSection>` components, as it is above in lines 74–76.

Reply:
I tried that, but I couldn't see the line showing up like that

Reply:
Ah well, that's awkward. I remember it wasn't yesterday, but it seems like it is now. Maybe something weird happened before or I was delusional 🤔